tue-common/rsettings_file

```diff
@@ -2,12 +2,13 @@
 
 # Robot environment
 #export ROBOT_ENV=reo2016
-export ROBOT_ENV=robotics_testlabs
+#export ROBOT_ENV=robotics_testlabs
```

Comment:
Default should be set to robotics_testlabs

Reply:
+1